apps/admin-gui/src/app/admin/admin-routing.module.ts

```diff
@@ -32,11 +32,13 @@ import { UserAccountsComponent } from '../users/pages/user-detail-page/user-acco
 import { AdminAuditLogComponent } from './pages/admin-page/admin-audit-log/admin-audit-log.component';
 import { AdminConsentHubsComponent } from './pages/admin-page/admin-consent-hubs/admin-consent-hubs.component';
 import { AdminSearcherComponent } from './pages/admin-page/admin-searcher/admin-searcher.component';
+import { RouteAuthGuardService } from '../shared/route-auth-guard.service';
 
 const routes: Routes = [
   {
     path: '',
     component: AdminPageComponent,
+    canActivateChild: [RouteAuthGuardService],
```

Comment:
Good point! Thanks to it I realized, that I need to add also `key.startsWith('identities')` to the `canNavigate` method for Perun admin section (due to route `admin/users/:userId/identities/:identityId` which will be accessible otherwise).